ironfish/src/rpc/routes/chain/getBlockInfo.ts

```diff
@@ -42,19 +62,41 @@ router.register<typeof GetBlockInfoRequestSchema, GetBlockInfoResponse>(
   GetBlockInfoRequestSchema,
   async (request, node): Promise<void> => {
     const hash = Buffer.from(request.data.hash, 'hex')
-    const header = await node.chain.getHeader(hash)
 
+    const header = await node.chain.getHeader(hash)
     if (!header) {
       throw new ValidationError(`No block with hash ${request.data.hash}`)
     }
 
+    const block = await node.chain.getBlock(header)
+    if (!block) {
+      throw new ValidationError(`No block with hash ${request.data.hash}`)
+    }
+
+    const transactions: GetBlockInfoResponse['block']['transactions'] = []
```

Comment:
For my own understanding, why is the type here in brackets? (I am assuming this is a TS thing I'm unfamiliar with.)

Reply:
If you have a type, you can reference inner types using brackets in Typescript without redeclaring them.

```typescript
type Foo = {
    bar: {
        baz: number
    }
}

type Baz = Foo['bar']['baz']
// Type of baz is number
```